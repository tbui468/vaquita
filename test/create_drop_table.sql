create database sol;
open sol;

create table planets (name string, moons int, gravity float, atmosphere bool);
drop table planets;

close sol;
drop database sol;
exit;
