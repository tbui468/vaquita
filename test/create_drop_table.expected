created database sol
opened database sol
created table planets
dropped table planets
closed database sol
dropped database sol
